examples/chart/teleport-cluster/templates/config.yaml

```diff
@@ -41,6 +41,7 @@ data:
     kubernetes_service:
       enabled: true
       listen_addr: 0.0.0.0:3027
+      kube_cluster_name: {{ required "kubeClusterName is required in chart values when kube role is enabled, see README" .Values.kubeClusterName }}
```

Comment:
I don't like the idea of making this a compulsory parameter and leaving it blank in the values file. Our AWS, GCP and custom guides for `telpeort-cluster` in the docs would also need to be updated to include this new value if we do this. You'll notice the chart linter is also failing because the value is now compulsory, which means this breaks backward compatibility.

Could we instead use a sane default (i.e. the same value as `clusterName`) if `kubeClusterName` isn't provided? This means that the installation commands all stay the same between v6 and v7, upgrades will work without anyone needing to add additional values and we don't need to modify any of the guides.

Reply:
@webvictim

> Could we instead use a sane default (i.e. the same value as clusterName) if kubeClusterName isn't provided?

 Agree, I have reworked it in that way. If `kubeClusterName` value is not provided the `clusterName` will be used as `kubeClusterName`